BedrockServer.cpp

```diff
@@ -1496,6 +1512,12 @@ void BedrockServer::_reply(unique_ptr<BedrockCommand>& command) {
     }
 
     command->response["nodeName"] = args["-nodeName"];
+    // This line will generate the node list that processed the command. 
+    // If the current node is on the right version of the cluster, then it will return the node itself
+    // If it's not on the right version, it will escalate to another follower
+    // The names will be generated in a way the the left-most node name is the last escalation step
+
+    command->response["nodesPath"] = string(command->response["nodesPath"]).empty() ? args["-nodeName"] : args["-nodeName"] +  "," + command->response["nodesPath"];
```

Comment:
What is this for? Just for the tests? I don't see why we need this.

Reply:
It was basically so we could track which server path a request does. I do use it in the test, and I couldn't think of another way to test if we were doing the desired path

Reply:
The test plugin does stuff like this. Implement a command in the test plugin, and use that in the tests. The command then can append it's own server path to the response each time it runs (or something similar).

Reply:
How do I get the nodeName other than from Args on the bedrock server?

Reply:
`args` is a public meber of the `BedrockServer`, I think you can just do something like `plugin().server.ags["nodeName"];`

Reply:
ok, I went on a different direction with the nodesPath, we should be good now!